test/utils/image/manifest.go

```diff
@@ -262,7 +263,9 @@ func initImageConfigs() map[int]Config {
 	configs[VolumeISCSIServer] = Config{e2eRegistry, "volume/iscsi", "2.0"}
 	configs[VolumeGlusterServer] = Config{e2eRegistry, "volume/gluster", "1.0"}
 	configs[VolumeRBDServer] = Config{e2eRegistry, "volume/rbd", "1.0.1"}
-	configs[WindowsNanoServer] = Config{e2eRegistry, "windows-nanoserver", "v1"}
+	// NOTE(claudiub): Do NOT change the value below to e2eRegistry, which is configurable. The image below
+	// needs to always be the one from gcr.io. (needed by: should be able to pull image from gcr.io)
+	configs[WindowsNanoServer] = Config{e2eGcRegistry, "windows-nanoserver", "v1"}
```

Comment:
can you just hardcode here directly if no plan to change to something else?

Reply:
sure.

Reply:
Done